package.json

```diff
@@ -1,28 +0,0 @@
-{
-  "name": "upaya",
-  "version": "0.0.1",
-  "private": true,
-  "scripts": {
-    "pa11y-ci": "pa11y-crawl --run \"bundle exec rails s\" --ci -w 10 --standard Section508 http://localhost:3000",
```

Comment:
we just want to remove the scripts, right? Not remove the entire `package.json?`